tests/integration/update.bats

```diff
@@ -175,7 +178,7 @@ EOF
     runc update test_update --pids-limit -1
     [ "$status" -eq 0 ]
     check_cgroup_value "pids.max" max
-    check_systemd_value "TasksMax" $SD_UNLIMITED
+    check_systemd_value "TasksMax" $SD_UNLIMITED || check_systemd_value "TasksMax" $SD_UNLIMITED_LEGACY
```

Comment:
Please fix the commit description, it currently says this systemd change only affects `MemoryLimit`, but apparently it is also for `TasksMax` and `MemorySwapMax` (at least).

Reply:
updated